Demo/src/main/java/com/braintreepayments/demo/PendingRequestStore.java

```diff
@@ -0,0 +1,52 @@
+package com.braintreepayments.demo;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import com.braintreepayments.api.PayPalPendingRequest;
+
+public class PendingRequestStore {
```

Comment:
This just an example of how a merchant could store these requests - could use Room or whatever other storage mechanism they want